app/controllers/saving_goals_controller.rb

```diff
@@ -0,0 +1,22 @@
+class SavingGoalsController < ApplicationController
+  def new
```

Comment:
When you end up with an empty action like this one you can delete it and rails will automatically render the corresponding view, provided you've created it.

In order words, if you delete this action, you'll still be able to visit the page to create a new Saving goal